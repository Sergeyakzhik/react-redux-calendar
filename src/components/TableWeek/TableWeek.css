.table-week td {
  height: 2rem;
  text-align: left;
  margin: 0;
  position: relative;
}

.table-week th:nth-child(2) {
  color: #ea0000;
}

.table-week td h4 {
  margin-top: 0rem;
  margin-bottom: 2rem;
  margin-left: 0.2rem;
}

.table-week {
  border-spacing: 0.15rem
}

.table-week .event {
  margin: 0;
  height: 100%;
  position: absolute;
  width: 100%;
  top: 0;
  border: 0.08rem solid black;
}

.table-week .event h4 {
  margin: 0;
  font-size: 0.8rem;
}

.table-week .events-list {
  top: 0;
  height: 100%;
}

.table-week .events-skeleton {
  position: absolute;
  width: 86.7%;
  height: 95.4%;
  top: 4.7rem;
  left: 9.3rem;
}

.table-week .events-skeleton table {
  table-layout: fixed;
  height: 100%;
}

.table-week .events-skeleton td {
  border: none;
  background-color: transparent;
  height: 15px;
}

.table-week .events-skeleton tbody td {
  text-align: center;
}
