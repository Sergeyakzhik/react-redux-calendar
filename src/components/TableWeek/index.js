import React from 'react';
import './style.css';

const TableWeek = props => (
  <div>
    <h2>{props.date}</h2>
    <table className="table-week">
      <thead>
        {props.tableHeader}
      </thead>
      <tbody>
        {props.tableRows}
      </tbody>
    </table>
  </div>
);

export default TableWeek;
