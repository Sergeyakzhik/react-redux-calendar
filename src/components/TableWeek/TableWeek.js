import React from 'react';
import './TableWeek.css';

let TableWeek = props => (
  <div>
    <h2>{props.date}</h2>
    <table className="table-week">
      <thead>
        {props.tableHeader}
      </thead>
      <tbody>
        {props.tableRows}
      </tbody>
    </table>
  </div>
);

export default TableWeek;
