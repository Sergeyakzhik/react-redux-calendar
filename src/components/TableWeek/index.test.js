import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16'

import TableWeek from './index';

Enzyme.configure({ adapter: new Adapter() })

describe('TableWeek', () => {
  let props = {
    tableRows: [],
    tableHeader: [],
    date: 'date'
  }

  let component = shallow(<TableWeek {...props} />);

  it('renders TableWeek component', () => {
    expect(component).toMatchSnapshot();
  })

  it('checks TableWeek date', () => {
    expect(component.find('h2').text()).toEqual('date');
  })
})
