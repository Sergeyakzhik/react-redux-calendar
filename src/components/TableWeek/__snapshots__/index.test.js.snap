// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableWeek render TableWeek component 1`] = `
<div>
  <h2>
    date
  </h2>
  <table
    className="table-week"
  >
    <thead />
    <tbody />
  </table>
</div>
`;
